tests/test_bukuDb.py

```diff
@@ -69,13 +69,33 @@ def vcr_cassette_dir(request):
     return os.path.join("tests", "vcr_cassettes", request.module.__name__)
 
 
+def rmdb(*bdbs):
+    for bdb in bdbs:
+        try:
+            bdb.cur.close()
+            bdb.conn.close()
+        except Exception:
+            pass
+    if exists(TEST_TEMP_DBFILE_PATH):
+        os.remove(TEST_TEMP_DBFILE_PATH)
+
+
 @pytest.fixture()
-def setup():
+def bukuDb():
     os.environ["XDG_DATA_HOME"] = TEST_TEMP_DIR_PATH
 
     # start every test from a clean state
-    if exists(TEST_TEMP_DBFILE_PATH):
-        os.remove(TEST_TEMP_DBFILE_PATH)
+    rmdb()
+
+    bdbs = []
+
+    def _bukuDb(*args, **kwargs):
+        nonlocal bdbs
+        bdbs += [BukuDb(*args, **kwargs)]
+        return bdbs[-1]
```

Comment:
Alternatively this could be implemented as a class with `BukuDb()` method, but this way seems less clunky.